acceptance/lib/puppet/acceptance/rpm_util.rb

```diff
@@ -15,10 +16,11 @@ def rpm_provider(agent)
 
       def setup(agent)
         cmd = rpm_provider(agent)
-        required_packages = ['createrepo', 'rpm-build']
+        required_packages = ['createrepo', 'curl', 'rpm-build']
```

Comment:
There is a larger question around whether or not Beaker should be upgrading `curl` - it performs a version check at the beginning of a run with https://github.com/puppetlabs/beaker/blob/8f660718f971c6663e28bf6b5cbf3d26157beae0/lib/beaker/host_prebuilt_steps.rb#L123-L124

Any thoughts @kevpl ?